src/app/clusters/webrtc-transport-provider-server/webrtc-transport-provider-server.cpp

```diff
@@ -223,12 +245,29 @@ uint16_t WebRTCTransportProviderServer::GenerateSessionId()
 // Command Handlers
 void WebRTCTransportProviderServer::HandleSolicitOffer(HandlerContext & ctx, const Commands::SolicitOffer::DecodableType & req)
 {
+    // If both the VideoStreamID and AudioStreamID are not present: Respond with CONSTRAINT_ERROR
+    if (!req.videoStreamID.HasValue() && !req.audioStreamID.HasValue())
+    {
+        ChipLogError(Zcl, "HandleSolicitOffer: Both video and audio streams missing");
+        ctx.mCommandHandler.AddStatus(ctx.mRequestPath, Protocols::InteractionModel::Status::ConstraintError);
```

Comment:
Followup issue (with comment pointing to it in the code) needed here tracking the spec fix in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11333, after which this will need to be an InvalidCommand status.

Reply:
Added, but I think this check should be completely removed after https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/11333